### PR TITLE
[enhancement] add allowBreak setting to antiAFK.

### LINKED ISSUE #1865: Disable allowBreak when AntiAfk is enabled
**Is your feature request related to a problem? Please describe.**
If you die and respawn it is possible for anti afk to break your bed. Eg you die and Anti-death-glitch-screen is active 

**Describe the solution you'd like**
While the movement part of anti afk is active there should be an option for a special clause making it impossible to break a bed.

 I know that #allowbreak exists but having something that automatically does it would make everything a lot less stressful.

**Describe alternatives you've considered**

**Additional context**
Personal experience

### PR DESCRIPTION
**Pull request by scorbett123**
Adds an option to disable allow break when antiAFK is enabled. 

Closes #1865

### CODE REVIEW

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt [hunk 1]
```diff
@@ -39,13 +39,15 @@ internal object AntiAFK : Module(
     private val walk = setting("Walk", true)
     private val radius by setting("Radius", 64, 8..128, 8)
     private val inputTimeout by setting("InputTimeout(m)", 0, 0..15, 1)
+    private val allowBreak by setting("AllowBreak", false)
```
Comment (l1ving) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt:42]: this should only be visible when the walk setting is on

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt [hunk 2]
```diff
@@ -39,13 +39,15 @@ internal object AntiAFK : Module(
...
 
     private var startPos: BlockPos? = null
     private var squareStep = 0
     private var baritoneDisconnectOnArrival = false
     private val inputTimer = TickTimer(TimeUnit.MINUTES)
     private val actionTimer = TickTimer(TimeUnit.TICKS)
     private var nextActionDelay = 0
+    private var wasAllowBreakAllowed = false
```
Comment (l1ving) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt:50]: this should follow the naming scheme of baritoneDisconnectOnArrival and move the var beside it as well

**Review by l1ving — CHANGES_REQUESTED** [review]

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt [hunk 3]
```diff
@@ -39,9 +39,11 @@ internal object AntiAFK : Module(
     private val walk = setting("Walk", true)
     private val radius by setting("Radius", 64, 8..128, 8)
     private val inputTimeout by setting("InputTimeout(m)", 0, 0..15, 1)
+    private val allowBreak by setting("AllowBreak", false, { walk.value })
```
Comment (l1ving) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt:42]: honestly in something like AntiAFK this should be called AllowBreakingBlocks

Comment (l1ving) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt:42]: sorry for all the requested changes

**Review by l1ving — CHANGES_REQUESTED** [review]

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt (hunk 3 above)
Comment (scorbett123) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt]: Don't worry, they are small changes that I can do quickly, and I understand that you want to keep the project as nice as possible, I would as well in your situation.

**Review by l1ving — APPROVED** [review]

**Comment by l1ving** [comment]
thanks!

**Comment by scorbett123** [comment]
Thanks for all the suggestions.

**Pull request merged**